airflow/models.py

```diff
@@ -1257,6 +1257,14 @@ def __init__(self, event, task_instance, owner=None, extra=None):
             self.execution_date = task_instance.execution_date
             task_owner = task_instance.task.owner
 
+        if 'task_id' in kwargs:
+            self.task_id = kwargs['task_id']
+        if 'dag_id' in kwargs:
+            self.dag_id = kwargs['dag_id']
+        if 'execution_date' in kwargs:
+            if kwargs['execution_date']:
+                self.execution_date = kwargs['execution_date'][:10]
```

Comment:
In the request it looks like 2015-01-01 00:00:00 (...) while in current log table it looks like just the first 10 characters (ISO). What would work?